README.md

```diff
@@ -1,38 +1,44 @@
-- [6-webpack-for-convenience](https://github.com/udacity/fend-webpack-content/tree/6-webpack-for-convenience)
+## First steps
 
-Each branch in this project is a step along the path to creating a fully functional webpack setup. In each branch, there will be a documentation file that lists out the steps taken in that branch (each step is also roughly a git commit if you look at the history) which you can use as a checklist when setting up your own projects. 
+### Install node modules
+Let's cache the node modules used by running `yarn install`.
 
-## What we will cover
+### Setting up Environment Variables
+You need to define a service reference to MeaningCloud. This is done via environment variables stored in the `.env` file. 
+As we don't want to share our secret api key, this file is included in the `.gitignore` so we don't push it by accident.
 
-We will cover:
+* Create a file called `.env`.
+* Add your data
+```
+API_KEY=[TOP_SECRET_API_KEY]
+```
 
-- Webpack entry point
-- Webpack output and dist folder
-- Webpack Loaders
-- Webpack Plugins
-- Webpack Mode
-- Tools for convenient Webpack development
+## Testing the App
+`yarn build-dev` will start up the backend (`node src/server/index.js`) and the frontend (`webpack serve --config webpack.dev.js --open`).  
+The last part (`--open`) will start a new tab in your browser and load the web app. The app will run on http://localhost:8080, 
+the backend on http://localhost:8081. 
 
-## Get Up and Running
+## Creating a Prod Distribution
+You can create a minified production distribution by simply running `yarn build-prod`. This will generate the minified 
+sources and pack it into the `dist` directory. If you want to run it on a server, just put the contents on the web root, 
+and you're done.
 
-Fork this repo, then clone the branch of your choice from your forked repo down to your computer:
+## Running the backend without generating the frontend
+If you simply want to fire up the backend (maybe you want to run postman against the endpoints?), run `yarn start`.  
+This will start the service on http://localhost:8081 for you. 
 
-```
-git clone -- git@github.com:[your-user-name]/webpack-express.git --
-```
+## Run the automated tests
+You can run the tests with `yarn test`. There is only one test at the moment, and it doesn't do much. But it's there at least.
+
+## Some last words ...
+The first part uses a sentiment analysis that I could not force seeing irony no matter how hard I tried. 
+I guess irony is still too much to ask an AI for.  
 
-`cd` into your new folder and run:
-- ```npm install```
-- ```npm start``` to start the app
-- this app runs on localhost:8080, but you can of course edit that in index.js
+Next to the Language Detection API, I'm using the summarization API to condense the article to some sentences. 
+The results are ... not the best possible. Don't shoot the messenger, I'm just fetching the API.
```

Comment:
I like your readme. It covers the basics, isn't too verbose and lets you get the project startet no matter if you want to test it locally or deploy it. Nice work!

Reply:
Thank you! 😄